values.yaml

```diff
@@ -198,6 +198,17 @@ server:
   # If not set to true, Vault server will not be installed. See vault.mode in _helpers.tpl for implementation details
   enabled: true
 
+  # [Enterprise Only] This value refers to a Kubernetes secret that you have
```

Comment:
Do we want to be a little more explicit here?  We could note that for enterprise Vault 1.8 and later, enterpriseLicense is required unless using the binaries with baked-in licenses (pro/prem) or if the cluster was created with an older Vault version and the license was stored.

I'm not sure that we have to, it's a genuine question.

Reply:
Good question; I think we can be more explicit about the cases where a license is required in the vault-helm [docs pages](https://www.vaultproject.io/docs/platform/k8s/helm), since setting `enterpriseLicense` here is just one way of setting the license on the vault pods.

Specifically I'm thinking of adding a page similar to https://www.consul.io/docs/k8s/installation/deployment-configurations/consul-enterprise.